moveit.rosinstall

```diff
@@ -24,3 +24,11 @@
     local-name: moveit_tutorials
     uri: https://github.com/ros-planning/moveit_tutorials.git
     version: master
+- git:
+    local-name: panda_moveit_config
+    uri: https://github.com/ros-planning/panda_moveit_config.git
+    version: melodic-devel
+- git:
+    local-name: moveit_resources
+    uri: https://github.com/rhaschke/moveit_resources.git
```

Comment:
Shouldn't that be https://github.com/ros-planning/moveit_resources?

Maybe I am missing a bit of MoveIt history here, but why duplicate the panda_moveit_config in moveit_resources if it needs to be kept in sync with the one in https://github.com/ros-planning/panda_moveit_config?

Reply:
The corresponding commit comment indicates this change as temporary. As long as https://github.com/ros-planning/moveit_resources/pull/28 is not yet merged and released, Travis would fail otherwise.
We maintain a separate moveit_resources package to resolve circular package dependencies due to MoveIt -> panda_moveit_config -> MoveIt. This PR exactly tries to resolve this circular dependency.

Reply:
Now that  ros-planning/moveit_resources#28 is merged, can we revert this?

Reply:
No, the release is not yet synced.

Reply:
Melodic release is synced, but Kinetic not yet.